web/public/mgmt/config.yml

```diff
@@ -3060,6 +3066,22 @@ collections:
       preview: false
     delete: false
     files:
+      - label: "Quick Action Icons"
+        name: "arrayOfQuickActionIcons"
+        file: "content/src/mappings/quickActionIcons.json"
+        fields:
+          - label: Quick Action Icons
+            name: arrayOfQuickActionIcons
+            widget: list
```

Comment:
dust -  maybe make this widget hidden and update the label so it informs content this collection is not editable